binding/ios/Leopard.swift

```diff
@@ -80,8 +106,12 @@ public class Leopard {
                 accessKey,
                 modelPathArg,
                 enableAutomaticPunctuation,
+                enableDiarization,
```

Comment:
you're missing the actual call to `pv_set_sdk` - double check that this is present in all your SDKs. If it's missing in one, it'll mess up our stats and we'll have to do a rerelease